front_end/src/app/admin/room/room.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+import { Router, ActivatedRoute, Params } from '@angular/router';
+import { AdminService } from '../admin.service';
+
+@Component({
+  selector: 'room',
+  templateUrl: 'room.component.html',
+  styleUrls: ['room.component.scss']
+})
+export class RoomComponent {
+  // room: any;
```

Comment:
yep - am using it in https://github.com/crdschurch/crds-signin-checkin/pull/29 (US5176)